### ISSUE TITLE
Bulk Data Files - 508 Issue 

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#2187]
## Required Fix 

- [x] RF-1
- [x] RF-2
- [x] RF-3

## Context 

| Issue # | Issue Title  | Issue Description  | How to Fix  | Additional Details  |
|---|---|---|---|---|
| RF-1 | Tab Order of the page is not logical | When tabbing through the page, the focus does not follow a logical order which may confuse users. This issue is a violation of Section 508 and WCAG 2.0 Success Criterion 2.4.3 | Modify the tab order so it is logical. Highly suggest following the visual order of the page. For further assistance see: W3 HTML Technique 4 W3 CSS Technique 27 | When the buttons on the page are accessed via keyboard, there is an extra tab stop, there should be only one. |
| RF-2 | Modal Dialog has incorrect Focus management | A modal dialogue appears that does not receive programmatic focus and is not immediately following that control that caused it to appear and/or focus is not returned when the dialog is closed. This issue is a violation of Section 508 and WCAG 2.0 Success Criterion 2.4.3 | When a modal Dialog Box appears, the content must either keyboard focus must be placed on the first actionable control. If there is no other actionable control, then focus should be placed on the close button. To ensure that the modal dialog operates correctly, code the dialog using ARIA. For further assistance see:  W3 Modal Specs Healthcare.gov Dialog W3 Failure 85 | When the Need help accessing the ftp site link is activated, focus is not put on the Close button. |
| RF-3 | Content for Screen Readers is hidden | There is informational content intended for screen reader users that is not intended to be displayed, but also is hidden because Display:none or aria hidden = true was used. This issue is a violation of Section 508 and WCAG 2.0 Success Criterion 1.3.1. | Remove display:none and/or aria hidden = true from the content. If hidden text is desired, use absolute positioning. For further assistance see: Webaim absolute positioning | On the FTP Help Modal, the Close and Done button have aria hidden=true:  <button type="button" class="usa-button usa-modal__close" data-testid="button" aria-controls="ftp-access-help-modal" aria-label="Close this window" data-close-modal="true"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 24 24" width="1em" class="usa-icon" focusable="false" role="img" aria-hidden="true"><path d="M0 0h24v24H0z" fill="none"></path><path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"></path></svg></button>  <button type="button" class="usa-button float-right" data-testid="button" aria-labelledby="close-ftp-access-help-modal" data-close-modal="true">Done</button> |

See [EPA CAMPD Bulk Data Files Accessibility Report.docx](https://github.com/US-EPA-CAMD/easey-ui/files/7414160/EPA.CAMPD.Bulk.Data.Files.Accessibility.Report.docx)

## Testing

Tester: @aprematta

**Comment by JanellC** [comment]
@aprematta This ticket is ready for 508 testing

**Comment by aprematta** [comment]
RF-1 and 2 pass. For RF-3, aria hidden was removed from the close and done buttons, however when the modal is active, there is this piece of code div id="root" aria-hidden="true" data-modal-hidden="". Since the modal is under this the entire content will be hidden to a screen reader.

**Comment by aprematta** [comment]
All issues pass, closing ticket
